exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 let roomMaxCapacity = 25;
-let enoughSpaceInRoom; // finish this statement
+let enoughSpaceInRoom = roomMaxCapacity >= studentCount + mentorCount; // finish this statement
```

Comment:
You are completely right Stephen, but when I save it with ctrl+s parenthesis is deleted by VSC.

Reply:
Hi Ekrem, 

It may be the pretty plugin for VS Code, I personally don't use this but if the plugin is preventing you from doing this please don't worry about it, your code is still valid :)